docs/src/operate/c8y/software-management.md

```diff
@@ -4,101 +4,172 @@ tags: [Operate, Installation, Software Management]
 description: Managing software on devices
 ---
 
-:::note
-As of now, this feature is supported only on devices with **debian** based
-distributions, which use the **apt** package manager(Ex: RaspberryPi OS , Ubuntu, Debian), from Cumulocity cloud.
+## Management
+
+%%te%% enables you to manage the software installed on your device. The following sections details how to install and remove software packages.
+
+:::info
+For more detailed information on the Cumulocity IoT **Device Management** features, please check out the [official documentation](https://cumulocity.com/docs/device-management-application/managing-device-data/#managing-software).
 :::
 
-Below steps show how to download, install and enable %%te%% software management feature.
+### To add a new software or software version {#add-to-repo}
 
-## Download and install software management packages on the device
+1. Navigate to the *Management* &rarr; *Software repository* page
 
-As a prerequisite, install [tedge and tedge-mapper](../../install/index.md) if not installed already.
+    ![Select software](images/software-repo-1-list.png)
 
-The %%te%% software management packages are in repository on GitHub: [%%te%%](https://github.com/thin-edge/thin-edge.io/releases).
+    The page will display all of the software packages which are are available for to be installed on a device.
 
-To download the package from github repository use the following command (use desired version):
+2. Select *Add software* and fill in the details
 
-```sh
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/<package>_<version>_<arch>.deb
-```
+    <p align="center">
+        <img width="40%" src={require('./images/software-repo-2-add.png').default} alt="Add software" />
+    </p>
 
-Where:
-- `version` -> %%te%% software management components version in x.x.x format
-- `arch` -> architecture type (amd64, armhf)
+    :::tip
+    * If you are using the `apt` software type and hosting the package is hosted in an external repository, then just use an a single space in the *Profile a file path* option.
```

Comment:
Either:

```suggestion
    * If you are using the `apt` software type and hosting the package in an external repository, then just use an a single space in the *Profile a file path* option.
```

Or:

```suggestion
    * If you are using the `apt` software type and the package is hosted in an external repository, then just use an a single space in the *Profile a file path* option.
```

Reply:
I chose the first option (but also merging @rina23q comment about the "Profile" instead of "Provide" typo.

Resolved by 1ef801db93367a3ddf2eaabd966fc77a4d88478e